vortex-fastlanes/src/bitpacking/mod.rs

```diff
@@ -71,8 +83,9 @@ impl BitPackedArray {
             encoded,
             validity,
             patches,
-            bit_width,
+            offset,
             len,
```

Comment:
wait... were bit_width and len incorrectly order swapped before????

Reply:
nah, this is by name not by position

Reply:
that's a nice Rust feature ❤️